src/Build/Evaluation/LazyItemEvaluator.IncludeOperation.cs

```diff
@@ -113,7 +113,11 @@ protected override ImmutableList<I> SelectItems(OrderedItemDataCollection.Builde
                             {
                                 using (_lazyEvaluator._evaluationProfiler.TrackGlob(_rootDirectory, glob, excludePatternsForGlobs))
                                 {
-                                    includeSplitFilesEscaped = EngineFileUtilities.GetFileListEscaped(_rootDirectory, glob, excludePatternsForGlobs, fileMatcher: FileMatcher);
+                                    (includeSplitFilesEscaped, FileMatcher.SearchAction action) = EngineFileUtilities.GetFileListEscaped(_rootDirectory, glob, excludePatternsForGlobs, fileMatcher: FileMatcher);
+                                    if (action == FileMatcher.SearchAction.ReturnLogDriveEnumerationWildcard)
+                                    {
+                                        _lazyEvaluator._loggingContext.LogWarning("InvalidAttributeValue", EscapingUtilities.UnescapeAll(glob), XMakeAttributes.include, XMakeElements.itemGroup);
```

Comment:
Test:
```
<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
  <ItemGroup>
    <FilesToCopy Include="$(Microsoft_WindowsAzure_EngSys)\**\*" Exclude="$(Microsoft_WindowsAzure_EngSys)\*.pdb;$(Microsoft_WindowsAzure_EngSys)\Microsoft.WindowsAzure.Storage.dll;$(Microsoft_WindowsAzure_EngSys)\Certificates\**\*" />
  </ItemGroup>
</Project>
```
Output:
![image](https://user-images.githubusercontent.com/89041446/144302167-ca40da93-e580-42c1-8fab-b377a9691bc3.png)

Reply:
Msbuild runs into this error both with and without my changes. I think the test case is faulty, but I'm not certain how to hit this codepath otherwise.

Reply:
I think this serves as a better test case:
```
<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
    <ItemGroup>
        <Compile Include="\**\*.cs"/>
    </ItemGroup>
    <Target Name="TestTarget">
        <ItemGroup>
            <Message Text="@(Compile->'%(Filename).cs')"/>
        </ItemGroup>
    </Target>
</Project>
```

Output (hits this LazyItemEvaluator codepath):
![image](https://user-images.githubusercontent.com/89041446/145131302-93835b5a-4b8b-48f8-8572-5f98ab265510.png)